indico/modules/events/registration/templates/management/email.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'layout/base.html' %}
+{% from 'forms/_form.html' import form_header, form_rows, form_footer  %}
+
+{% block title %}
+    {% trans %}Send email{% endtrans %}
+{% endblock %}
+
+{% block subtitle %}
+    {% trans %}to selected people{% endtrans %}
```

Comment:
OK, I just repeated the same login as in the agreements. Will check it out, thx.